### DISCUSSION TITLE
Storage Persistence

### DISCUSSION THREAD

**Discussion by DevSecDude** [cn-terraform/terraform-aws-sonarqube#22] [Q&A] [answered]
How is the storage being persisted for container running in fargate? Specifically the following:

      - sonarqube_data:/opt/sonarqube/data
      - sonarqube_extensions:/opt/sonarqube/extensions
      - sonarqube_logs:/opt/sonarqube/logs

**Comment by jnonino** [comment, marked as answer]
Hi, right n ow there is no mechanism to persist that data. One idea could be to implement EFS drive so containers can start, stop and restart mounting the same drive.
